libethash-cl/ethash_cl_miner.cpp

```diff
@@ -222,21 +237,18 @@ bool ethash_cl_miner::searchForAllDevices(unsigned _platformId, function<bool(cl
 
 void ethash_cl_miner::doForAllDevices(function<void(cl::Device const&)> _callback)
 {
-	vector<cl::Platform> platforms;
-	cl::Platform::get(&platforms);
+	vector<cl::Platform> platforms = getPlatforms();
 	if (platforms.empty())
-	{
-		ETHCL_LOG("No OpenCL platforms found.");
 		return;
-	}
 	for (unsigned i = 0; i < platforms.size(); ++i)
 		doForAllDevices(i, _callback);
 }
 
 void ethash_cl_miner::doForAllDevices(unsigned _platformId, function<void(cl::Device const&)> _callback)
 {
-	vector<cl::Platform> platforms;
-	cl::Platform::get(&platforms);
+	vector<cl::Platform> platforms = getPlatforms();
+	if (platforms.empty())
```

Comment:
Thanks for noticing. Refactoring makes you so blind to obvious things like that.